lib/src/hpack/hpack.dart

```diff
@@ -167,9 +167,10 @@ class HPackDecoder {
 //    - without huffman encoding
 //    - without using the dynamic table
 class HPackEncoder {
+  // ignore: unused_field
   int _maxHeaderTableSize = 4096;
 
-  final IndexTable _table = new IndexTable();
+  // final IndexTable _table = new IndexTable();
```

Comment:
We shouldn't comment out or ignore these. At the point where we add code that would require this it will be obvious to add it back.